documentation/latest/dev/installation.md

```diff
@@ -109,14 +109,14 @@ make run
 For registering devices, it's important to download certs and set correct
 permission:
 
-~~~
+```shell
```

Comment:
I changed it in this PR which has a screenshot of the output:
https://github.com/project-flotta/project-flotta.github.io/pull/52

Did you check how it is rendered on the local site? if this looks good, so I'm okay with this change.
If problem remains, let's stay with `~~~` instead.